apps/explorer/src/components/transactions/index.tsx

```diff
@@ -79,6 +79,7 @@ export function Transactions({
         <div>
             {recentTx ? (
                 <TableCard
+                    noFullWidth
```

Comment:
We discussed this in a meeting but I don't think this is the approach that we want to take